com.github.rigellute.spotify-tui.yaml

```diff
@@ -0,0 +1,28 @@
+command: spt
+id: com.github.rigellute.spotify-tui
+separate-locales: false
+finish-args:
+  - "--share=network"
+  - "--filesystem=home"
```

Comment:
What does an streaming client need full access to home for?

Reply:
The configuration file for the client is created at ${HOME}/.config/spotify-tui/client.yml.

Reply:
Then that is a bug that should be fixed upstream. It should respect `$XDG_CONFIG_HOME` being set.

Reply:
I'll see if I can make it work with the $XDG_CONFIG_HOME